### ISSUE TITLE
ability to use private media

### ISSUE THREAD

**Issue by chrisdrackett** [spalt/EO1#6]
I would like to display images of my family on the EO1, but don't want to add or make these images public. It would be great if we could access via the API private photos. I assume this would require auth-ing to the API

**Comment by spalt** [comment]
@chrisdrackett looking at the flickr API docs it seems a bit of a pain to implement private search, but I can see what I can do.  For me, using "private tags" is the best solution currently.

**Comment by chrisdrackett** [comment]
shouldn't a call with authentication give access to private photos? I'm seeing that when I "sign call as Chris Drackett" I'm able to get back photos marked as private in my stream.

My issue is that I don't want to have photos of my kids public online before they can make their own decision about what they want public or not.

**Comment by spalt** [comment]
yep, understood.  when EO1 was actually working, it had a similar issue....  I saw lots of random kids show up on my device, probably without people realizing it.   I'll dig into it over the next week or two and see what it would take to implement the authentication.

**Comment by aphex3k** [comment]
I created a fork of this repo that works with self-hosted Immich instead public flickr. 

https://gitea.codingmerc.com/michael/EO1
